nHentai_downloader/nHentai_downloader.user.js

```diff
@@ -35,6 +37,12 @@ jQuery(function ($) {
     var outputExt = 'cbz'; // or 'zip'
 
     /**
+     * Put images in subfolder inside archive
+     * @type {Boolean}
+     */
+    var subfolder = false;
```

Comment:
```javascript
getInfo().split('\n')[0]
```
This is manga title. You can use the function **Extract to [directory]**  of file decompressor like 7zip.
Or it will be combined with `extractedInfo` as directory name to classify when unpacking?